### PR TITLE
upgrade to polkadot-sdk-1.3.0

### PR DESCRIPTION
**Pull request by brenzi**
* upgrade polkadot-sdk deps to v1.3
* bump major crate versions to 3.0.0 (published)
* toml cleanup with cargo zepter

### CODE REVIEW

**Comment by brenzi** [comment]
published like this. fixing CI issues now...

File: Cargo.lock [hunk 1]
```diff
@@ -1616,51 +1816,51 @@ dependencies = [
...
 ]
 
 [[package]]
 name = "encointer-ceremonies-assignment"
-version = "2.0.0"
+version = "3.0.1"
 dependencies = [
  "encointer-primitives",
- "sp-core 24.0.0",
+ "sp-core",
```
Comment (clangenb) [inline Cargo.lock:1842]: Oh, nice it seems that we already had duplicates before, and this is fixed now.

File: balances-tx-payment/rpc/Cargo.toml [hunk 2]
```diff
@@ -20,17 +20,20 @@ parking_lot = "0.12.0"
...
-pallet-transaction-payment-rpc = { default-features = false, version = "26.0.0" }
-sc-rpc = "25.0.0"
-sc-rpc-api = "0.29.0"
-sp-api = "22.0.0"
-sp-blockchain = "24.0.0"
-sp-core = "24.0.0"
-sp-rpc = "22.0.0"
-sp-runtime = "27.0.0"
+pallet-transaction-payment = { default-features = false, version = "25.0.0" }
+pallet-transaction-payment-rpc = { default-features = false, version = "27.0.0" }
```
Comment (clangenb) [inline balances-tx-payment/rpc/Cargo.toml:29]: I think `default-features = false` is not correct here.

File: balances-tx-payment/rpc/Cargo.toml [hunk 3]
```diff
@@ -20,17 +20,20 @@ parking_lot = "0.12.0"
...
+sc-rpc = "26.0.0"
+sc-rpc-api = "0.30.0"
+sp-api = "23.0.0"
+sp-blockchain = "25.0.0"
+sp-core = "25.0.0"
+sp-rpc = "23.0.0"
+sp-runtime = "28.0.0"
+
+[features]
+std = []
```
Comment (clangenb) [inline balances-tx-payment/rpc/Cargo.toml:39]: Was this added by zepter? Not sure if I agree that it should be here, when a crate can't compile to `no_std`.

File: bazaar/Cargo.toml [hunk 4]
```diff
@@ -13,41 +13,48 @@ codec = { package = "parity-scale-codec", version = "3.6.4", default-features =
...
+    "frame-benchmarking",
+    "frame-support/runtime-benchmarks",
+    "frame-system/runtime-benchmarks",
+]
 
-try-runtime = ["frame-system/try-runtime"]
+try-runtime = [
+    "encointer-communities/try-runtime",
+    "frame-system/try-runtime",
+
```
Comment (clangenb) [inline bazaar/Cargo.toml:59]: Interesting that taplo accepts this empty line in the array.

File: ceremonies/Cargo.toml [hunk 5]
```diff
@@ -13,61 +13,76 @@ codec = { package = "parity-scale-codec", version = "3.6.4", default-features =
...
-runtime-benchmarks = ["frame-benchmarking", "sp-application-crypto"]
+runtime-benchmarks = [
+    "encointer-balances/runtime-benchmarks",
+    "encointer-communities/runtime-benchmarks",
+    "encointer-scheduler/runtime-benchmarks",
+    "frame-benchmarking",
+    "frame-support/runtime-benchmarks",
+    "frame-system/runtime-benchmarks",
+    "sp-application-crypto",
+
```
Comment (clangenb) [inline ceremonies/Cargo.toml:77]: empty line

File: ceremonies/Cargo.toml [hunk 6]
```diff
@@ -13,61 +13,76 @@ codec = { package = "parity-scale-codec", version = "3.6.4", default-features =
...
+]
+
+try-runtime = [
+    "encointer-balances/try-runtime",
+    "encointer-communities/try-runtime",
+    "encointer-scheduler/try-runtime",
+    "frame-system/try-runtime",
 
-try-runtime = ["frame-system/try-runtime"]
+]
```
Comment (clangenb) [inline ceremonies/Cargo.toml:86]: empty line

**Review by clangenb — COMMENTED** [review]
Nice, looks good to me! Just some questions, and minor comments.

File: balances-tx-payment/rpc/Cargo.toml (hunk 3 above)
Comment (brenzi) [reply balances-tx-payment/rpc/Cargo.toml:39]: yes, probably zepter added it because there was the above mentioned no-default-features present

**Pull request merged**